playbooks/roles/katello_provisioning/tasks/libvirt.yml

```diff
@@ -0,0 +1,57 @@
+---
+- name: 'install the virtualization hypervisor package group'
+  yum:
+    name: "@Virtualization Hypervisor"
+    state: present
+
+- name: 'install the virtualization tools package group'
+  yum:
+    name: "@Virtualization Tools"
+    state: present
+
+- name: 'disable libvirt authentication'
+  lineinfile:
+    dest: /etc/libvirt/libvirtd.conf
+    line: 'auth_unix_rw = "none"'
+
+- name: 'restart libvirt'
+  service: name=libvirtd state=restarted
+
+- name: 'create the provisioning libvirt network'
+  virt_net:
+    command: define
+    name: provision
+    xml: '{{ lookup("template", "../templates/provision.xml.j2") }}'
+
+# Setup TFTP for libvirt
```

Comment:
It needs to be done, because of a bit of a chicken and egg problem.  You can't bring the libvirt network up if /var/lib/tftpboot doesn't exist, but you can't run the installer unless the virbr1 exists.

```
 Evaluation Error: Error while evaluating a Function Call, Could not get the ip address from fact ipaddress_virbr1 at /usr/share/foreman-installer/modules/foreman_proxy/manifests/proxydhcp.pp:8:5 on node centos7-katello-p4-nightly.example.com
```

I could run the installer twice, once with tftp enable, and then again later with DHCP, but I'd rather not, it's too time consuming especially with Katello.